templates/config/config.sh.aws

```diff
@@ -0,0 +1,49 @@
+# Amazon Web Services
+# Kubernetes cluster creation definitions, generated by ./configure.
+# It's OK to modify this file as necessary to customize your deployment,
+# but note it may be overwritten if ./configure is run again.
+
+# Cloud hosting provider
+PROVIDER=aws				# Cloud hosting provider
+USERNAME=                   # The username associated with the current AWS profile (REQUIRED)
+
+# Kubernetes cluster definitions
+CLUSTER_NAME=genesis-test			# Kubernetes cluster name
+K8S_VERSION=1.14			# Kubernetes engine version
+SIZE=m5.xlarge			# Node type to use for the k8s cluster
+NODES=3					# Number of nodes in the k8s cluster
+NODE_POOL_NAME=\${SIZE//./-}			# Name of the k8s node pool, replace '.' in name with '-' for safety with AWS nodegroups
+REGION=us-west-2				# AWS region where to create the k8s cluster
+CONTEXT=\$USERNAME@\$CLUSTER_NAME.\$REGION.eksctl.io	# Name of the k8s context for the cluster
+DASHBOARD_VERSION=v2.0.0-beta6		# The version of Kubernetes Dashboard to install
+
+# JupyterHub version to install, and kubernetes namespace to install it into
+JHUB_VERSION=0.9.0-alpha.1.028.00bc15c	# JupyterHub Helm chart version
+JHUB_HELM_RELEASE=genesis-hub		# The name of the installed release
+JHUB_K8S_NAMESPACE=genesis-hub		# The k8s namespace to install into
+
+# JupyterHub image configuration
+HUB_IMAGE=mjuric/hub:gallifrey.local-20191125172436		# custom JupyterHub image (e.g. foo/hub:v0.3)
```

Comment:
Update these to latest images on master in `config.sh.do` (they support Kubernetes 1.16). Also update `JHUB_HELM_RELEASE` and `JUPYTER_IMAGE`.

N.b., we should split off provider-independent options from provider-specific options, so there's no duplication (but that can be done on a different PR, after this one is merged).

Reply:
Updated `JHUB_VERSION` and `HUB_IMAGE`, the rest look the same as what's on the master branch right now.